card-unification.md

```diff
@@ -0,0 +1,90 @@
+# Card Unification
+
+  * [ ] Change card shape to object with `type`, `name`, `render` and optional `edit` properties
+  * [ ] Card's `type` is validated by renderer (dom renderer cannot render 'text', e.g.)
+  * [ ] Change arguments passed by editor to card's `render` (or `edit`) method
+    * [ ] single argument object with `name`, `buffer`, `isInEditor`, `options`, `editorEnv` and `payload` properties
```

Comment:
Ah ok, I think I had the entire meaning of `isInEditor` wrong. Seems ok, I'm not sure what a better name would be :+1: 

`editorEnv` has some limited items like `name` for cards not rendered by the editor.